src/controllers/jobSeeker.ts

```diff
@@ -129,17 +124,3 @@ export const updateJobSeeker = async (
     next(new NotFoundError('ID NOT FIND'))
```

Comment:
This should be `InternalServerError` instead since we already handle not found error above